libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -705,7 +705,7 @@ - (void)handleAppWillResignActive:(NSNotification *)notification
     // If we make the snapshot window the active window now, that's where the SFAuthenticationSession's view controller will end up
     // Then when the application is foregrounded and the snapshot window is dismissed, we will lose the SFAuthenticationSession
     SFSDKWindowContainer* activeWindow = [SFSDKWindowManager sharedManager].activeWindow;
-    if (([activeWindow isAuthWindow]  && ![activeWindow.topViewController isKindOfClass:[SFLoginViewController class]]) ||  [activeWindow isPasscodeWindow]) {
```

Comment:
Why drop the `&& ![activeWindow.topViewController isKindOfClass:[SFLoginViewController class]]`?

It came with:
https://github.com/forcedotcom/SalesforceMobileSDK-iOS/commit/af1152d8d387191fa07c2891f0a3762d85e918d9

Reply:
@wmathurin was noticing some problems with snapshots over the authwindow. Removing this check disallows the the snapshot  from being rendered over auth window.

Reply:
That check was to only disabling snapshot over SFAuthenticationSession - so you are saying it's also causing issues with regular login.